config/tileset/elevation.json

```diff
@@ -12,470 +12,547 @@
       "name": "waimea-dam_2023_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/hamilton_2019/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/hamilton_2019_dem_1m/01HZ67AF6SVAD4SNNMZ28K974V/",
+      "minZoom": 9,
       "title": "Waikato - Hamilton LiDAR 1m DEM (2019)",
       "name": "hamilton_2019_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/hamilton_2023/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/hamilton_2023_dem_1m/01HZ67AXVJ9N35PFNBE5HX6KFW/",
+      "minZoom": 9,
       "title": "Waikato - Hamilton LiDAR 1m DEM (2023)",
       "name": "hamilton_2023_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/huntly_2015-2019/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/huntly_2015-2019_dem_1m/01HZ67C43AKY44WM7R0YKA3ZDF/",
+      "minZoom": 9,
       "title": "Waikato - Huntly LiDAR 1m DEM (2015-2019)",
       "name": "huntly_2015-2019_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/reporoa-and-upper-piako-river_2019/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/reporoa-and-upper-piako-river_2019_dem_1m/01HZ67C4MCA93ZQ93RH99DQ872/",
+      "minZoom": 9,
       "title": "Waikato - Reporoa and Upper Piako River LiDAR 1m DEM (2019)",
       "name": "reporoa-and-upper-piako-river_2019_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/thames_2017-2019/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/thames_2017-2019_dem_1m/01HZ67CB7CEZX0JYWJ38N3920E/",
+      "minZoom": 9,
       "title": "Waikato - Thames LiDAR 1m DEM (2017-2019)",
       "name": "thames_2017-2019_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/waikato_2021/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/waikato_2021_dem_1m/01HZ67GB12GXJDBF92Q4E2T9A2/",
+      "minZoom": 9,
       "title": "Waikato LiDAR 1m DEM (2021)",
       "name": "waikato_2021_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/waikato/west-coast-and-hauraki-plains_2015/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/west-coast-and-hauraki-plains_2015_dem_1m/01HZ67CKSE7VXYT4Q71SAW3PR5/",
+      "minZoom": 9,
       "title": "Waikato - West Coast and Hauraki Plains LiDAR 1m DEM (2015)",
       "name": "west-coast-and-hauraki-plains_2015_dem_1m"
     },
     {
-      "minZoom": 9,
+      "2193": "s3://nz-elevation/wellington/wellington_2013-2014/dem_1m/2193/",
       "3857": "s3://linz-basemaps/elevation/3857/wellington_2013-2014_dem_1m/01HZ67MBQJ8VASM6Z69PMATPHH/",
+      "minZoom": 9,
       "title": "Wellington LiDAR 1m DEM (2013-2014)",
       "name": "wellington_2013-2014_dem_1m"
     },
     {
-      "minZoom": 9,
       "3857": "s3://linz-basemaps/elevation/3857/hutt-city_2021_dem_1m/01HZ67GF9ZANVBZW5PEM98V0T7/",
```

Comment:
Is Hutt City missing due to the geojson problem?

Reply:
This should be fixed now.